lib/serve.js

```diff
@@ -129,88 +127,89 @@ function createHandler(dir, static, timeout) {
 
     var isBase64 =
       request.body &&
-      !(request.headers["content-type"] || "").match(
-        /text|application|multipart\/form-data/
+      !(request.headers['content-type'] || '').match(
+        /text|application|multipart\/form-data/,
       );
     var lambdaRequest = {
       path: request.path,
       httpMethod: request.method,
       queryStringParameters: queryString.parse(request.url.split(/\?(.+)/)[1]),
       headers: request.headers,
       body: isBase64
-        ? Buffer.from(request.body.toString(), "utf8").toString("base64")
+        ? Buffer.from(request.body.toString(), 'utf8').toString('base64')
         : request.body,
-      isBase64Encoded: isBase64
+      isBase64Encoded: isBase64,
     };
 
     var callback = createCallback(response);
 
     var promise = handler.handler(
       lambdaRequest,
       { clientContext: buildClientContext(request.headers) || {} },
-      callback
+      callback,
     );
 
     var invocationTimeoutRef = null;
 
     Promise.race([
       promiseCallback(promise, callback),
-      new Promise(function(resolve) {
-        invocationTimeoutRef = setTimeout(function() {
+      new Promise(function (resolve) {
+        invocationTimeoutRef = setTimeout(function () {
           handleInvocationTimeout(response, timeout);
           resolve();
         }, timeout * 1000);
-      })
+      }),
     ]).then(
-      result => {
+      (result) => {
         clearTimeout(invocationTimeoutRef);
         return result; // not used, but writing this to avoid future footguns
       },
-      err => {
+      (err) => {
         clearTimeout(invocationTimeoutRef);
         throw err;
-      }
+      },
     );
   };
 }
 
-exports.listen = function(port, static, timeout) {
+exports.listen = function (port, isStatic, timeout) {
```

Comment:
For some reason `jest` tries to transpile `static` as a reserved keyword which caused the `serve` test to fail.
Instead of messing about with jest config files, just renamed the argument.